src/api/Feed.js

```diff
@@ -661,19 +682,20 @@ class Feed extends Base {
      *
      * @param {BoxItem} file - The file to which the comment belongs to
      * @param {string} commentId - Comment ID
-     * @param {BoxItemPermission} permissions - Permissions for the comment
+     * @param {BoxCommentPermission} permissions - Permissions for the comment
+     * @param {boolean} shouldUseThreadedComments - Indicator for using ThreadedComments api
      * @param {Function} successCallback - the function which will be called on success
      * @param {Function} errorCallback - the function which will be called on error     *
      * @return {void}
      */
     deleteComment = (
         file: BoxItem,
         commentId: string,
-        permissions: BoxItemPermission,
+        permissions: BoxCommentPermission,
+        shouldUseThreadedComments?: boolean,
```

Comment:
This is a breaking change for this method, since parameter order is important. Do we use this API anywhere else? Would creating a new set of methods for the threaded comments be cleaner?

Reply:
We are using this API only in `ActivitySidebar` and I've updated it to support the addition of the `shouldUseThreadedComments` parameter, but I agree, it's safer to split all methods related to comment operations since they're using a different API endpoints.